reana_job_controller/kubernetes_job_manager.py

```diff
@@ -118,6 +118,10 @@ def __init__(
             for job.
         :type rucio: bool
         """
+        # we only need this for kubernetes
+        cmd += " >> /opt/app/log.log 2>&1; sleep 10; rm -f /opt/app/running"
```

Comment:
If using `trap` instead of just appending the `rm` command, code needs to be changed in order to allow using multiple commands as traps as Kerberos already adds a trap, and multiple traps for one signal are not allowed.